src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/owner/PocketPeripheralOwner.java

```diff
@@ -1,12 +1,14 @@
 package de.srendi.advancedperipherals.common.addons.computercraft.owner;
 
 import dan200.computercraft.api.pocket.IPocketAccess;
+import de.srendi.advancedperipherals.AdvancedPeripherals;
```

Comment:
I can't leave comments on lines that haven't changed, but `getLevel`/`getPos` should `IPocketAccess.getLevel`/`IPocketAccess.getPosition` instead.